spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe IssuesController, type: :controller do
+  describe 'GET #new' do
+    it 'when user is not a reporter' do
+      get 'new'
+      expect(response).not_to redirect_to new_issue_path
+    end
+
+    it 'when user is a reporter' do
+      user = create(:user, role: :reporter)
```

Comment:
@andrjuha01  ah, sorry, my bad. 

But it is maybe better use here [traits](https://robots.thoughtbot.com/remove-duplication-with-factorygirls-traits) here and let instead of local variables http://betterspecs.org/#let .